antora/ec-policies-antora-extension/index.js

```diff
@@ -229,41 +219,55 @@ const helpers = {
 module.exports.register = function() {
   this.on('contentAggregated', async ({ contentAggregate }) => {
 
-    const rawAnnotationsData = await helpers.readRegoAnnotations()
-    const pipelineAnnotations = helpers.processAnnotationsData(rawAnnotationsData, "policy.pipeline")
-    const releaseAnnotations = helpers.processAnnotationsData(rawAnnotationsData, "policy.release")
-
-    // Skip modules other than the the ec-policies module
-    contentAggregate.filter(c => c.name == 'ec-policies').forEach(content => {
-      // Import yaml
-      const yaml = this.require('js-yaml')
+    // Only operate on the ec-policies docs
+    const content = contentAggregate.find(c => c.name == 'ec-policies')
 
-      // Find and load the acceptable bundle data
-      const rawBundlesData = helpers.dataFromAntoraFile(yaml, content, 'acceptable_tekton_bundles.yml')
-      const acceptableBundles = helpers.processBundlesData(rawBundlesData)
+    //------------------------------------------------------------------
+    // Extract annotations from the rego files
+    //
+    const regoFiles = helpers.filesMatching(content, /^policy\/.*(?!_test)\.rego$/)
+    const opaInspectPromises = regoFiles.map(f => opa.inspect(f.path, f._contents.toString()))
```

Comment:
This should work with the latest opa-inspect-js, I think:

```suggestion
    const regoFiles = helpers.filesMatching(content, /^policy\/.*(?!_test)\.rego$/).map(f => f.path)
    const opaInspectPromises = opa.inspect(regoFiles)
```